src/main/resources/mappings/anomaly-results.json

```diff
@@ -107,26 +107,56 @@
     "model_id": {
       "type": "keyword"
     },
-    "totalUpdates": {
-      "type": "long"
-    },
     "startOfAnomaly": {
       "type": "boolean"
     },
     "inHighScoreRegion": {
       "type": "boolean"
     },
-    "relativeIndex": {
-      "type": "integer"
+    "approx_anomaly_start_time": {
+      "type": "date",
+      "format": "strict_date_time||epoch_millis"
```

Comment:
can you explain why it can be `strict_date_time` or `epoch_millis`?

Reply:
I am following other date like execution_end_time